src/sections/Blog/Related-NewsBlogs/relatedNewsBlogs.style.js

```diff
@@ -0,0 +1,155 @@
+import styled from "styled-components";
```

Comment:
It'd be great to reuse as much of this as possible from the implementation in on the home page. Are these styles different?

Reply:
Can you create this as a component in which only the required data would be passed as props? This component can the be reused in Related Blogs and News

Reply:
Yes Correct .. I will do that .:+1: